content/php/concepts/math-functions/terms/max/max.md

```diff
@@ -0,0 +1,73 @@
+---
+Title: 'Max()'
+Description: 'Used to find the highest value in an array or a list of variables. It returns the highest value from the input provided.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Design'
+  - 'Web Development'
+Tags:
+ - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`max()`** function in PHP is used to find the highest value in an [array](https://www.codecademy.com/resources/docs/php/arrays) or a list of [variables](https://www.codecademy.com/resources/docs/php/variables). It returns the highest value from the input provided.
+
+## Syntax
+
+```pseudo
+max($var1, $var2, $var3, ...);
+```
+
+`$var1`, `$var2`, `$var3` are the [variables](https://www.codecademy.com/resources/docs/php/variables) or [array](https://www.codecademy.com/resources/docs/php/arrays) from which the maximum value will be returned.
+
+It can be called either with multiple parameters or with a single array parameter.
+
+When called with multiple parameters, the **`max()`** function takes a comma-separated list of variables or values, like this:
+
+```php
+$max_value = max($var1, $var2, $var3);
+
+```
```

Comment:
This is already mentioned in the previous syntax block. Maybe you would like to replace this with an array

Reply:
Or remove this section because it looks like the following portion speaks on having a single array as the parameter